### ISSUE TITLE
Can't locate object method "new" via package "Bio::DB::GFF::Typename"

### ISSUE THREAD

**Issue by eserte** [bioperl/Bio-DB-SeqFeature#1]
Most of my smoker systems started to show the following test failure:
```
...
Can't locate object method "new" via package "Bio::DB::GFF::Typename" at /home/cpansand/.cpan/build/2019022316/Bio-DB-SeqFeature-1.7.3-nBy0uI/blib/lib/Bio/DB/SeqFeature/Store/memory.pm line 679.
# Looks like you planned 116 tests but ran 20.
# Looks like your test exited with 2 just after 20.
t/SeqFeature.t ......... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 96/116 subtests 
...
```

**Comment by cjfields** [comment]
There is a missing check for the `eval` statement there; I suspect that the Bio-DB-GFF distribution isn't pulled in somehow for tests.  Interestingly enough it's also not listed as a dependency either; I suspect it's due to the implicit way that code is loaded in.

**Comment by cjfields** [comment]
Testing a fix with the new release (addressed in eeb33aa8fc9f1b2bdd2df9b563ff8bb5452ffbcc).  The issue is that `Bio::DB::GFF::Typename` was missed by `dzil` when finding dependencies as it was loaded dynamically.  I'm unsure why (I suspect this was a temporary solution which has now become entrenched as tests require it now).
